Codigo/modHechizos.bas

```diff
@@ -4118,9 +4120,11 @@ Public Sub DesplazarHechizo(ByVal UserIndex As Integer, ByVal Dire As Integer, B
 128                 TempHechizo = .Stats.UserHechizos(CualHechizo)
 130                 .Stats.UserHechizos(CualHechizo) = .Stats.UserHechizos(CualHechizo + 1)
 132                 .Stats.UserHechizos(CualHechizo + 1) = TempHechizo
-
+                    SpellInterval = .Counters.UserHechizosInterval(CualHechizo)
```

Comment:
This line can be deleted, you assign the same value as in line 4097 ?

Reply:
we do, but they are on different branch, if we move this to the top over the branch we will miss the array bound test that depends on the direction, i think its better to leave it as it is